frontend/scripts/router/nav-links.js

```diff
@@ -50,10 +50,10 @@ export default {
         payload: { section: 'faq' }
       }
     },
-    // {
-    //   name: 'Team',
-    //   page: 'team'
-    // },
```

Comment:
Are we sure about this?

Reply:
I'm sure we need to revert this before merging ;)

Reply:
Hmm, dunno when that should be enabled, maybe you are right that not today :D, will comment this out again.